proxy/proxy/handler.go

```diff
@@ -107,10 +114,22 @@ func (h *Handler) sendHTTPSProxyStatusBadGateway(c net.Conn) (err error) {
 	return util.WriteWithValidation(c, []byte("HTTP/1.1 501 Bad Gateway\r\n\r\n"))
 }
 
+var (
+	HttpOkByteSize         = uint64(len([]byte("HTTP/1.1 200 OK\r\n\r\n")))
```

Comment:
const httpTunnelMade string = "HTTP/1.1 200 OK\r\n\r\n"
var httpTunnelMadeSize = balabla

Reply:
do NOT ever made 2 copies of the same string, which made lead several unexpected bugs